/*                             Throw Exception */
/* Used For Throwing Custom Exception  */
public class ThrowDemo {
    public static void main(String[] args) {

        // Explicitly Worked By User
        throw new ArithmeticException("throw by us / by zero ");
    }
}
